cypress/support/auto_login.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+if (Cypress.env('SECURITY_ENABLED')) {
+  Cypress.Cookies.debug(false);
+
+  beforeEach(() => {
+    cy.login();
+  });
```

Comment:
we've had discussions as a team about how we want to remove the usage of `beforeEach` whenever possible, due to the following:

- The tests were taking too long
- The crashes that were happening due to the duration of the tests

Would a future implementation where this is replaced with `before` work?

Reply:
Due to test isolation, this needs to be in a `bewforEach` block.